### PR TITLE
Checkin temp fix

### PR DESCRIPTION
**Pull request by Sam-Park**

### CODE REVIEW

File: src/data/dummyData.js [hunk 1]
```diff
@@ -0,0 +1,111 @@
+const monthlyData = {
+    "ok": true,
+    "error": null,
+    "data": {
+        "2 Sat": [
```
Comment (jackzhen1996) [inline src/data/dummyData.js:5]: is `2 Sat` the format we're gonna be getting from the API ?

File: src/components/Calendar.js [hunk 2]
```diff
@@ -9,31 +9,48 @@ import {Box} from './previewEntries'
...
+     *Try to find today's checkIn in the checkin array by using of 'L' format dates ('L' means 'local time')
+     */
+    const todayJournal = []
+    // const todayJournal = journals.find(journal => moment(journal.journals.timestamp).format('L') === moment(today).format('L'))
+    const todaysDate =  moment(today).format('D');
+   
+    // iterate over the checkinObject to find if the current day is in the data and if it is push it to todayJournal
+        for (const prop in checkinObject['data']){    
+            if (prop.charAt(0) == todaysDate){
+                if(checkinObject['data'][prop].length > 1){
```
Comment (jackzhen1996) [inline src/components/Calendar.js:42]: i think we can combine line 41 and 42 into a single statement

File: src/components/checkinDetails.js [hunk 3]
```diff
@@ -1,11 +1,12 @@
-import React, { useEffect, useState } from "react";
+import React, { useEffect, useState, setState } from "react";
```
Comment (jackzhen1996) [inline src/components/checkinDetails.js:1]: are we using `setState` anywhere?

File: src/components/checkinDetails.js [hunk 4]
```diff
@@ -25,72 +26,99 @@ const moodMap = {
...
-  );
-  let specificIndex = entry.journals.indexOf(specificTime);
-
-  //Navigating from monthly preview: set initial index to zero
-  if (!time) {
-    specificIndex = 0;
+  // console.log("TIME", time)
+  // console.log("allentries%%%%%%%%%%%%%%%%%%%%%%%%%%%%", allEntries)
+  // console.log("entry###########################", entry)
+  // console.log("sprite@@@@@@@@@@@@@@@@@@@@@@@@@", spriteActivityData)
```
Comment (jackzhen1996) [inline src/components/checkinDetails.js:39]: do we need the comments from line 36 to line 39?

File: src/components/previewEntries.js [hunk 5]
```diff
@@ -27,31 +28,36 @@ export const Box = ({
...
+  //     <Text style={styles.category}>
+  //       {journal.substring(0, lastCommaIndex + 1)}{" "}
+  //       <Text style={{ fontWeight: "400" }}>and</Text>{" "}
+  //       {journal.substring(lastCommaIndex + 1)}
+  //     </Text>
+  //   );
+  // }
+ 
+  // console.log('FILTEREDDAYS################', filteredDays[filteredDays.length-1])
+  // capitalize function for mood as mood comes back as lowercase from Database
```
Comment (jackzhen1996) [inline src/components/previewEntries.js:57]: probably don't need the comments from line 35 to 57

**Review by jackzhen1996 — CHANGES_REQUESTED** [review]
Good work getting the check-in calendar to work! i might need some more time to review this since its a pretty big change, but can you also remove all the unused comments and `console.logs` ?

File: src/data/dummyData.js (hunk 1 above)
Comment (Sam-Park) [reply src/data/dummyData.js:5]: yes this is the data structure coming back from monthly checkins get request
image

**Comment by Sam-Park** [comment]
made changes requested, thanks Jack!

**Review by jackzhen1996 — APPROVED** [review]

**Pull request merged**